eng/pipelines/templates/pipeline.yml

```diff
@@ -88,6 +88,7 @@ resources:
 
 variables:
   DOTNET_NOLOGO: 1
+  NUGET_EXPERIMENTAL_CHAIN_BUILD_RETRY_POLICY: 3,1000
```

Comment:
I'm just curious how this number was selected?

Reply:
It was used as the example when the PR added the feature: https://github.com/NuGet/NuGet.Client/pull/4259